Unix/installbuilder/datafiles/Linux.data

```diff
@@ -189,6 +190,40 @@ ConfigureOmiService() {
     ${{SERVICE_CTL}} start
 }
 
+ConfigureLogRotate()
+{
+    echo "Configure log rotate for OMI"
+    # create the logrotate file if it doesn't exist
+    if [ ! -f /etc/logrotate.d/omi ]; then
+        cat /etc/opt/omi/conf/omilogrotate.conf > /etc/logrotate.d/omi
+    fi
+}
```

Comment:
Is it necessary to copy this to another file?  It's just an extra file to keep track of.

Reply:
It is a standard way for log rotate configuration file, thanks. I followed what scx and oms done.

Reply:
OK.  do scx and oms only log rotate on Linux platforms only?

Reply:
yes.